### ISSUE TITLE
Accessibility issues on the links

### ISSUE THREAD

**Issue by Kvaibhav01** [dailydotdev/docs#59]
On the docs website, when dark mode is turned on, all the links have a gray color with a black background. This makes for a bad contrast between the text and it is bad for accessibility for people with low vision.

I checked it on the dev tools and here's what I got:

<img width="1454" alt="Dev tools screenshot showing low contrast ratio" src="https://user-images.githubusercontent.com/11731837/153145303-2bcb8bec-eb26-4caf-92ff-5e64cbda1487.png">

The contrast fails on both the recommended WCAG levels: AA and AAA. The contrast ratio is significantly low at 3.9.

The minimum acceptable ratio comes as 5.72:

![A screenshot showing minimum acceptable contrast ratio](https://user-images.githubusercontent.com/11731837/153145696-9449c2ef-5730-4e95-a435-b12001820291.png)

And to pass both the levels we need it to be around 7.11:

![A screenshot showing minimum acceptable contrast ratio for both levels](https://user-images.githubusercontent.com/11731837/153145859-fd9aedfb-3f2d-4b30-98d2-c5b7487a743c.png)

I tried to dig into the codebase to change the color code myself but it seems that it uses the default `prism-react-renderer` theme as described in the _docusaurus.config.js_ file:

`const darkCodeTheme = require('prism-react-renderer/themes/dracula');`

I don't know how to change it then because I guess the `color: hsl(1deg 1% 63%);` property is coming from that package only. 

Please fix it so that the accessibility improves and it's better to read.

**Comment by GrahamTheDev** [comment]
This should now be resolved with version 2 of the docs, hopefully we have now fixed this long standing issue!

Thanks so much for raising it.
